arcgis-ios-sdk-samples/Features/Display feature layers/DisplayFeatureLayersViewController.swift

```diff
@@ -0,0 +1,223 @@
+        setFeatureLayer(featureLayer, viewpoint: viewpoint)
+    }
+    
+    /// Load a feature layer with a portal item.
+    func loadPortalItem() {
+        // Set the portal.
+        let portal = AGSPortal.arcGISOnline(withLoginRequired: false)
+        // Create the portal item with the item ID for the Portland tree service data.
+        let item = AGSPortalItem(portal: portal, itemID: "1759fd3e8a324358a0c58d9a687a8578")
+        // Create the feature layer with the item and layer ID.
+        let featureLayer = AGSFeatureLayer(item: item, layerID: 0)
+        // Set the viewpoint to Portland, Oregon.
+        let viewpoint = AGSViewpoint(latitude: 45.5266, longitude: -122.6219, scale: 6e3)
+        setFeatureLayer(featureLayer, viewpoint: viewpoint)
+    }
+    
+    /// Load a feature layer with a local geodatabase.
+    func loadGeodatabase() {
+        if let geodatabase = geodatabase {
+            getFeatureLayer(from: geodatabase)
+        } else {
+            // Instantiate the geodatabase with the file name.
+            geodatabase = AGSGeodatabase(name: "LA_Trails")
+            
+            // Load the geodatabase for feature tables.
+            geodatabase.load { [weak self] (error: Error?) in
+                guard let self = self else { return }
+                if let error = error {
+                    self.presentAlert(error: error)
+                } else {
+                    self.getFeatureLayer(from: self.geodatabase)
+                }
+            }
+        }
+    }
+    
+    /// Load a feature layer with a local geopackage.
+    func loadGeopackage() {
+        if let geoPackage = geoPackage {
+            getFeatureLayer(from: geoPackage)
+        } else {
+            // Create a geopackage from a named bundle resource.
+            geoPackage = AGSGeoPackage(name: "AuroraCO")
+            
+            // Load the geopackage.
+            geoPackage.load { [weak self] (error: Error?) in
+                guard let self = self else { return }
+                if let error = error {
+                    self.presentAlert(error: error)
+                } else {
+                    self.getFeatureLayer(from: self.geoPackage)
+                }
+            }
+        }
+    }
+    
+    /// Load a feature layer with a local shapefile.
+    func loadShapefile() {
+        // Create a shapefile feature table from a named bundle resource.
+        let shapefileTable = AGSShapefileFeatureTable(name: "ScottishWildlifeTrust_ReserveBoundaries_20201102")
+        
+        // Create a feature layer for the shapefile feature table.
+        let featureLayer = AGSFeatureLayer(featureTable: shapefileTable)
+        // Set the viewpoint to Scotland.
+        let viewpoint = AGSViewpoint(latitude: 56.641344, longitude: -3.889066, scale: 6e6)
+        setFeatureLayer(featureLayer, viewpoint: viewpoint)
+    }
+    
+    func getFeatureLayer(from geodatabase: AGSGeodatabase) {
```

Comment:
The prefix `get` makes it sound like the method is returning something, which it isn't. A better name might be `setFeatureLayer(from:)`.

Similar with `getFeatureLayer(from:)` -> `setFeatureLayer(from:)`.